activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -14,10 +14,12 @@ def accept(o)
           send m, o
         end
 
-        delegate :quote_column_name, :quote_table_name, :quote_default_expression, :type_to_sql,
-          :options_include_default?, :supports_indexes_in_create?, :supports_foreign_keys?, :foreign_key_options, to: :@conn
-        private :quote_column_name, :quote_table_name, :quote_default_expression, :type_to_sql,
-          :options_include_default?, :supports_indexes_in_create?, :supports_foreign_keys?, :foreign_key_options
+        delegate :quote_column_name, :quote_table_name, :quote_default_expression, :quote, :type_to_sql,
+          :options_include_default?, :supports_indexes_in_create?, :supports_foreign_keys?, :foreign_key_options,
+          :supports_comments?, :supports_comments_in_create?, to: :@conn
```

Comment:
Removed `:supports_comments_in_create?` and `:supports_comments?` from `SchemaCreation`.